src/octotree.js

```diff
@@ -123,17 +138,17 @@ $(document).ready(() => {
 
     function tryLoadRepo(reload) {
       hasError = false;
-      const remember = store.get(STORE.REMEMBER);
-      const shown = store.get(STORE.SHOWN);
       const token = store.get(STORE.TOKEN);
+      const pinned = store.get(STORE.PINNED);
 
       adapter.getRepoFromPath(currRepo, token, (err, repo) => {
         if (err) {
           showError(err);
         } else if (repo) {
           $toggler.show();
 
-          if (remember && shown) {
+          if (pinned) {
+            togglePin(true);
             toggleSidebar(true);
```

Comment:
Maybe togglePin() should also show sideBar when true? Make the method simpler to use and reason about.